modules/apps/27-interchain-accounts/types/errors.go

```diff
@@ -4,6 +4,12 @@ import (
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 )
 
+const (
+	// AcknowledgementError defines a string constant included in error acknowledgements
+	// NOTE: Changing this const is state machine breaking as acknowledgements are written into state
+	AcknowledgementError = "error handling packet on destination chain: see events for details"
```

Comment:
Maybe `host` instead of `destination`?

```suggestion
	AcknowledgementError = "error handling packet on host chain: see events for details"
```

Reply:
Sure, makes sense for the context of ICA!